### PR TITLE
DOC:642: Document new toolbar_persist setting

### PR DESCRIPTION
**Pull request by SimonFc**
Related Ticket: TINY-4847

Description of Changes:

- Documented the new `toolbar_persist` setting added by TINY-4847.

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] `_data/nav.yml` has been updated (if applicable)
- [x] Files has been included where required (if applicable)
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: _includes/configuration/toolbar-persist.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+## toolbar_persist
+
+Use this option to disable the automatic show and hide behaviour of the inline toolbar. Enabling this setting would, by default, always show the inline toolbar and not hide it whenever focus is lost.
+
+**Type:** `Boolean`
```
Comment (lnewson) [inline _includes/configuration/toolbar-persist.md:5]: We'll need to add the `{{requires_5_5v}}` note here at some stage, but I don't believe @tylerkelly13 has set that up yet.

**Review by lnewson — APPROVED** [review]

File: _includes/configuration/toolbar-persist.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
```
Comment (tylerkelly13) [inline _includes/configuration/toolbar-persist.md:6]:
```suggestion

**Default Value:** `false`

```

File: _includes/configuration/toolbar-persist.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## toolbar_persist
+
+Use this option to disable the automatic show and hide behaviour of the inline toolbar. Enabling this setting would, by default, always show the inline toolbar and not hide it whenever focus is lost.
```
Comment (tylerkelly13) [inline _includes/configuration/toolbar-persist.md:3]:
```suggestion
This option disables the automatic show and hide behavior of the inline toolbar and menu bar. Enable this option to always show the inline toolbar and menu bar and not hide them whenever focus is lost. Use this option to make the inline toolbar and menu bar persistent.
```

**Review by tylerkelly13 — APPROVED** [review]
Looks Good To Me.

**Review by ltrouton — APPROVED** [review]

File: _includes/configuration/toolbar-persist.md (hunk 3 above)
Comment (metricjs) [reply _includes/configuration/toolbar-persist.md:3]: `Enable this option to always show the toolbar and menu bar, and...` <- added comma? Overall wording feels a bit... repetitive?

**Review by metricjs — APPROVED** [review]

**Review by lnewson — APPROVED** [review]

**Review by EkimChau — APPROVED** [review]

**Pull request merged**